cmd/gateway/commands.go

```diff
@@ -450,3 +457,29 @@ func createSleepCommand() *cobra.Command {
 
 	return cmd
 }
+
+func parseFlags(flags *pflag.FlagSet) (flagKeys, flagValues []string) {
+	flagKeys = []string{}
```

Comment:
style:
we typically declare slices as `var a []int` in the code.

optimization:

can you figure out the length of slice (is there any method to call to get the length) so that we follow this https://github.com/nginxinc/nginx-gateway-fabric/blob/main/docs/developer/go-style-guide.md#construct-slices-with-known-capacity ?

Reply:
>  I'm leaning towards letting it be as it is currently

👍